fluentd/configs.d/filter/max_priority.conf

```diff
@@ -0,0 +1,4 @@
+<filter journal>
```

Comment:
could the file be "max-priority.conf" for consistency?

Would actually be clearer to me if it were named "exclude-journal-debug.conf"